client/my-sites/plugins/mailpoet-confirmation/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { Card, Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import DocumentHead from 'calypso/components/data/document-head';
+import FormattedHeader from 'calypso/components/formatted-header';
+import Main from 'calypso/components/main';
+import { addMailPoetUpgrade } from 'calypso/data/marketplace/mailpoet-add-upgrade';
+import { errorNotice, successNotice } from 'calypso/state/notices/actions';
+import './style.scss';
+
+export const MailPoetConfirmationPage = ( { siteId }: { siteId: number } ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+	const [ isBusy, setIsBusy ] = useState( false );
+	const [ isCompleted, setIsCompleted ] = useState( false );
+
+	const getItNowClickHandler = async () => {
+		setIsBusy( true );
+		try {
+			await addMailPoetUpgrade( siteId );
+
+			dispatch(
+				successNotice(
+					translate(
+						'The MailPoet is added to your site. MailPoet plugin will be installed and activated shortly.'
```

Comment:
```suggestion
						'MailPoet Business is added to your site. MailPoet plugin will be installed and activated shortly.'
```

Reply:
Changed in 019b1b5a1bc223848ae169124e6c2a675dcc2ec